airflow/jobs.py

```diff
@@ -2383,9 +2371,8 @@ def _collect_errors(self, ti_status, session=None):
 
         return err
 
-    @provide_session
     def _execute_for_run_dates(self, run_dates, ti_status, executor, pickle_id,
-                               start_date, session=None):
+                               start_date):
```

Comment:
Nit, please collapse this into a single line

Reply:
fixed